test/suite/test_verbose04.py

```diff
@@ -0,0 +1,136 @@
+          
+        # Close the initial connection. We will be opening new connections with different verbosity
+        # settings throughout this test.
+        self.close_conn()
+
+        # Test passing a single verbose category, 'all' along with the verbosity level
+        # WT_VERBOSE_DEBUG_1 (1). Ensuring the only verbose output generated is related to the
+        # 'all' category.
+        with self.expect_verbose(['all:1'], all_verb, self.is_json) as conn:
+            # Perform a set of simple ALL operations to generate verbose ALL messages.
+            uri = 'table:test_verbose04_all'
+            session = conn.open_session()
+            session.create(uri, self.collection_cfg)
+            c = session.open_cursor(uri)
+            c['all'] = 'all'
+            c.close()
+            session.close()
+            
+        # At this time, there is no verbose messages with the category WT_VERB_ALL and the verbosity
+        # level WT_VERBOSE_INFO (0), hence we don't expect any output.
+        with self.expect_verbose(['all:0'], all_verb, self.is_json, False) as conn:
+            uri = 'table:test_verbose04_all'
+            session = conn.open_session()
+            session.create(uri, self.collection_cfg)
+            c = session.open_cursor(uri)
+            c['all'] = 'all'
+            c.close()
+            session.close()
+            
+        # Test passing another single verbose category, 'all' with different verbosity levels.
+        # Since there are verbose messages with the category WT_VERB_COMPACT and the verbosity
+        # levels WT_VERBOSE_INFO (0) through WT_VERBOSE_DEBUG_5 (5), we can test them all.
+        cfgs = ['all:0', 'all:1', 'all:2', 'all:3', 'all:4', 'all:5']
+        for cfg in cfgs:
+            with self.expect_verbose([cfg], all_verb, self.is_json) as conn:
+                # Create a simple table to invoke compaction on. We aren't doing anything
+                # interesting with the table, we want to simply invoke a compaction pass to generate
+                # verbose messages.
+                uri = 'table:test_verbose04_all'
+                session = conn.open_session()
+                session.create(uri, self.collection_cfg)
+                session.compact(uri)
+                session.close()
+                
+    # Test use cases passing multiple verbose categories, ensuring we only produce verbose output
+    # for specified categories.
+    def test_verbose_multiple(self):
+        self.close_conn()
+        # Test passing multiple verbose categories, being 'api' & 'all' & 'version' with different dedicated
+        # verbosity levels to each category. Ensuring the only verbose output generated is related
+        # to those two categories.
+        cfgs = ['api:0,all:1,version:0', 'version:0,all,api:0']
+        
+        #all_verb_except_api_and_version contains all verb flag(except WT_VERB_API & WT_VERB_VERSION)
```

Comment:
Can we try to reuse the previously defined variables with all the categories but removing it the desired categories from it?

Reply:
@etienneptl 
 I have made some changes, please reviiew the new code again.

thanks.